test/test_prototype_transforms_functional.py

```diff
@@ -1337,3 +1346,42 @@ def _compute_expected_bbox(bbox, output_size_):
         else:
             expected_bboxes = expected_bboxes[0]
         torch.testing.assert_close(output_boxes, expected_bboxes)
+
+
+def test_correctness_center_crop_segmentation_mask_on_fixed_input(device):
+    mask = torch.ones((1, 6, 6), dtype=torch.long, device=device)
+    mask[:, 1:5, 2:4] = 0
+
+    out_mask = F.center_crop_segmentation_mask(mask, [2])
+    expected_mask = torch.zeros((1, 4, 2), dtype=torch.long, device=device)
+    torch.testing.assert_close(out_mask, expected_mask)
+
+
+@pytest.mark.parametrize("output_size", [[4, 3], [4]])
+def test_correctness_center_crop_segmentation_mask(output_size):
+    def _compute_expected_segmentation_mask():
+        _output_size = output_size if isinstance(output_size, tuple) else (output_size, output_size)
+
+        _, h, w = mask.shape
+        left = w - _output_size[0]
+        top = h - _output_size[1]
+
+        return mask[:, top : _output_size[1], left : _output_size[0]]
+
+    mask = torch.randint(0, 2, shape=(1, 6, 6))
+    actual = F.center_crop_segmentation_mask(mask, output_size)
+
+    expected = _compute_expected_segmentation_mask()
+    assert expected == actual
+
+
+@pytest.mark.parametrize("output_size", [[4, 3], [4], [7, 7]])
+@patch("torchvision.prototype.transforms.functional._geometry.center_crop_image_tensor")
+def test_correctness_center_crop_segmentation_mask_mock(center_crop_mock, output_size):
+    mask, expected = Mock(spec=torch.Tensor), Mock(spec=torch.Tensor)
+    center_crop_mock.return_value = expected
```

Comment:
We can use the [`mocker`](https://pytest-mock.readthedocs.io/en/latest/usage.html) fixture here:

```suggestion
def test_correctness_center_crop_segmentation_mask_mock(mocker, center_crop_mock, output_size):
    mask = mocker.MagicMock(spec=torch.Tensor)
    expected = mocker.MagicMock(spec=torch.Tensor)
    mocker.patch("torchvision.prototype.transforms.functional._geometry.center_crop_image_tensor", return_value=expected)
```

Note that you have to install `pytest-mock` for it to be available. We are doing that in CI:

https://github.com/pytorch/vision/blob/3ec4b94998bd2b413f1a5c77bfed590ce684b9b9/.github/workflows/prototype-tests.yml#L40